### ISSUE TITLE
Program crashes with no error message when trying to launch an istance (or update) when the disk is full

### ISSUE THREAD

**Issue by zibasPk** [MultiMC/Launcher#5352]
### Operating System

Windows

### Description of bug

The whole program crashes when trying to launch an istance or update when the disk is full.
Not a huge issue but has left me confused on why I couldn't start any istance, an error message would be apreciated.

### Steps to reproduce

- have the multiMC folder on an almost full disk (in my case it was more or less 20Mb of free space)
- Open the program and try launching an istance or try updating

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by Zetabite** [comment]
Multimc is preventing you to do stupid things. Trying to write data on an (almost) full disk, is stupid. Free up data or get a new drive. Even a temporary storage would do it for now. If you would play with the instance, you would surely corrupt a world you play, you wouldnt want that.

**Comment by FlyingBaguett3** [comment]
I agree that it should at least give an error message before closing so the user knows what the issue is.

**Comment by zibasPk** [comment]
> Multimc is preventing you to do stupid things. Trying to write data on an (almost) full disk, is stupid. Free up data or get a new drive. Even a temporary storage would do it for now. If you would play with the instance, you would surely corrupt a world you play, you wouldnt want that.

Sure its great that it stops you from doing it. I'm only saying that catching the issue and displaying a popup message instead of closing the application would be a much better User experience. People often don't realize they have a full drive, so if you don't tell them that's the reason why the application closed they might simply assume it's broken.
